_posts/2022-04-04-grant-announcements.md

```diff
@@ -0,0 +1,182 @@
+---
+title: "Announcing Brink's Latest Round of Developer Grants"
+permalink: /blog/2022/04/04/grantees/
+layout: post
+author: brink
+name: "brink"
+alt: Brink announces latest round of developer grants
+category: "News"
+description: Brink announces the latest round of developer grants.
+---
+
+Brink is proud to announce the recipients of its latest round of open source developer grants.
+After an open grant application process, the [Brink board of directors][brink
+board] has selected four developers who have demonstrated continued commitment to
+open source Bitcoin development. The grant recipients are:
```

Comment:
Nitpick: I think the repeated "X writes of his Bitcoin experience and goals" can be expressed universally here, e.g. change "The grant recipients are:" to "Each of the four grant recipients describes their Bitcoin experience and goals below:"